webassembly/Dockerfile

```diff
@@ -1,4 +1,24 @@
-FROM dockcross/web-wasm:20240529-0dade71 AS base
+FROM ubuntu:25.04 AS base
```

Comment:
this is not a LTS version. Doesnt work with 24.04 ?

Reply:
Nop, according to Jaswant I need very recent CMake/Ninja versions only available in 25.04